assets/scss/local.scss

```diff
@@ -28,7 +28,7 @@
   display: inline-block;
 }
 
-.overview-page-goal-summary-counts {
+.summary-counts-grid {
```

Comment:
Renamed these css classes to make them a little less overview page and goal centric; as I'm using the same classes for the layout for the session counts 👍